### PR TITLE
Dependencies

### PR DESCRIPTION
**Pull request by dolezel**
Some test are still failing and also when used in storybook, though when I use it in my project it is working.
There library used, which also needs to be upgraded, but that repo is half-abandoned :/
https://github.com/azuqua/react-dnd-scrollzone/pull/38

I will try to look into it more, or if someone have time/idea - improvements are welcome :)

closes: #363 
closes: #336

### CODE REVIEW

**Comment by dolezel** [comment]
Fixed, though I need to depend on my https://github.com/dolezel/react-dnd-scrollzone which is fixing react-dnd in way compatible with this library

**Comment by dolezel** [comment]
Fixes #363
Fixes #336

**Comment by wuweiweiwu** [comment]
wow @dolezel Thanks for the huge amount of work on this!

File: package.json [hunk 1]
```diff
@@ -64,9 +60,9 @@
   "dependencies": {
     "lodash.isequal": "^4.5.0",
     "prop-types": "^15.6.1",
-    "react-dnd": "3.0.2",
-    "react-dnd-html5-backend": "3.0.2",
-    "react-dnd-scrollzone": "^4.0.0",
+    "react-dnd": "^6.0.0",
+    "react-dnd-html5-backend": "^6.0.0",
+    "react-dnd-scrollzone": "github:dolezel/react-dnd-scrollzone.git#41203ef00c991f744aa87e897d7d518d32f6a994",
```
Comment (wuweiweiwu) [inline package.json]: Do you think in the future you can publish your own package? maybe a merge request to fix that library?

**Comment by wuweiweiwu** [comment]
Would you like to become a maintainer of this package?

**Comment by dolezel** [comment]
True to say, I didn't look much on how internals work. I just played with tooling on many of my projects so I have good grasp how these libs work and stick together.
But I can help with some tasks

File: package.json (hunk 1 above)
Comment (dolezel) [reply package.json]: I submitted PR, but do not expect it will be merged. 🤷‍♂️

**Comment by wuweiweiwu** [comment]
@dolezel awesome! ill add you as a collaborator

**Pull request merged**